src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -509,6 +515,117 @@ export default class AircraftCommander {
 
     /**
      * @for AircraftCommander
+     * @method runSayAltitude
+     * @param aircraft
+     * @return {array} [success of operation, readback]
+     */
+    runSayAltitude(aircraft) {
+        const altitude = _round(aircraft.altitude, -2);
+        const isClimbingOrDescending = aircraft.trend !== 0;
+        const readback = {};
+        let altitudeChangeString = '';
+
+        if (isClimbingOrDescending) {
+            altitudeChangeString = 'leaving ';
+        } else {
+            altitudeChangeString = 'at ';
+        }
+
+        readback.log = `${altitudeChangeString}${altitude}`;
+        readback.say = `${altitudeChangeString}${radio_altitude(altitude)}`;
+
+        return [true, readback];
+    }
+
+    /**
+     * @for AircraftCommander
+     * @method runSayAssignedAltitude
+     * @param aircraft
+     * @return {array} [success of operation, readback]
+     */
+    runSayAssignedAltitude(aircraft) {
+        const altitude = _round(aircraft.mcp.altitude, -2);
+        const readback = {};
+
+        readback.log = `assigned ${altitude}`;
+        readback.say = `assigned ${radio_altitude(altitude)}`;
+
+        return [true, readback];
+    }
+
+    /**
+     * @for AircraftCommander
+     * @method runSayHeading
+     * @param aircraft
+     * @return {array}	[success of operation, readback]
+     */
+    runSayHeading(aircraft) {
+        const heading = _round(radiansToDegrees(aircraft.heading));
+        const readback = {};
+
+        readback.log = `heading ${heading}`;
+        readback.say = `heading ${radio_heading(heading)}`;
+
+        return [true, readback];
+    }
+
+    /**
+     * @for AircraftCommander
+     * @method runSayAssignedHeading
+     * @param aircraft
+     * @return {array}	[success of operation, readback]
+     */
+    runSayAssignedHeading(aircraft) {
+        if (aircraft.mcp.headingMode !== MCP_MODE.HEADING.HOLD) {
+            return [false, `we haven't been assigned a heading`];
```

Comment:
Since there is no use of inline variable references, and this is actually a simple string, we should be using single quotes instead of back ticks. I believe lint should be showing a warning for this as well.